crates/nu-protocol/src/shell_error.rs

```diff
@@ -51,8 +51,8 @@ pub enum ShellError {
         #[label("value originates from here")] Span,
     ),
 
-    #[error("Pipeline mismatch.")]
-    #[diagnostic(code(nu::shell::pipeline_mismatch), url(docsrs))]
+    #[error("Only supports for specific types.")]
```

Comment:
I'd like it if this had "input types" instead of just "types" to differentiate it from argument type errors.

Reply:
Agreed :-D Add it